modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -3,10 +3,8 @@
 // * installing/installing_bare_metal/installing-bare-metal.adoc
 // * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
 // * installing/installing_ibm_z/installing-ibm-z.adoc
-// * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
-// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
-// * installing/installing_ibm_z/installing-ibm-power.adoc
-// * installing/installing_ibm_z/installing-restricted-networks-ibm-power.adoc
```

Comment:
Perhaps a merge conflict gone wrong? These four IBM Z assemblies still all include this module.

Reply:
This diff change should only be including the `installing-rhv restricted-network.adoc` assembly addition. Your latest update added the `installing-ibm-power.adoc` assembly back, but the other three (`installing-restricted-networks-ibm-z.adoc`, `installing-ibm-z-kvm.adoc`, `installing-restricted-networks-ibm-power.adoc`) should also be added back.